buffer.py

```diff
@@ -1025,47 +1053,57 @@ def toggle_read_mode(self):
     @interactive
     def scroll_up(self):
         self.update_vertical_offset(min(self.scroll_offset + self.scroll_step, self.max_scroll_offset()))
+        self.clear_synctex_indicator()
 
     @interactive
     def scroll_down(self):
         self.update_vertical_offset(max(self.scroll_offset - self.scroll_step, 0))
+        self.clear_synctex_indicator()
 
     @interactive
     def scroll_right(self):
         self.update_horizontal_offset(max(self.horizontal_offset - self.scroll_step, (self.rect().width() - self.page_width * self.scale) / 2))
+        self.clear_synctex_indicator()
 
     @interactive
     def scroll_left(self):
         self.update_horizontal_offset(min(self.horizontal_offset + self.scroll_step, (self.page_width * self.scale - self.rect().width()) / 2))
+        self.clear_synctex_indicator()
 
     @interactive
     def scroll_up_page(self):
         # Adjust scroll step to make users continue reading fluently.
         self.update_vertical_offset(min(self.scroll_offset + self.rect().height() - self.scroll_step, self.max_scroll_offset()))
+        self.clear_synctex_indicator()
 
     @interactive
     def scroll_down_page(self):
         # Adjust scroll step to make users continue reading fluently.
         self.update_vertical_offset(max(self.scroll_offset - self.rect().height() + self.scroll_step, 0))
+        self.clear_synctex_indicator()
```

Comment:
After synctex_pos_y become global coordiante, we just need test in paintEvent, don't need write clear_synctex_indicator.